integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -31,6 +34,7 @@ def __init__(
         collection_name: str = "documents",
         embedding_function: str = "default",
         persist_path: Optional[str] = None,
+        distance_function: DistanceFunction = "l2",
```

Comment:
```suggestion
        distance_function: Literal["l2", "cosine", "ip"] = "l2",
```

I would suggest we simply go with a Literal here.